src/wm_i3.rs

```diff
@@ -55,39 +54,39 @@ fn crawl_windows(root_node: &Node, workspace: &Workspace) -> Result<Vec<DesktopW
         for node in &nodes_to_explore {
             next_vec.extend(node.nodes.iter());
             next_vec.extend(node.floating_nodes.iter());
-            if node.window.is_some() {
```

Comment:
Why did you get rid of this check?

Reply:
I checked to see if happened during my manual merge, but I see the change in @JayceFayne's original PR, so I'll let them answer. 

https://github.com/svenstaro/wmfocus/pull/72/files#diff-d3be31b5118184c4b41438972fe1789134adf6111fe59d7b9ad18d24ecc59296L57

Reply:
@svenstaro What does the check do? I've been using @JayceFayne's sway branch for months and haven't noticed a problem.

Reply:
Guess I should have documented that because I honestly don't remember. It either has to do with an empty root window without children or with non-tiling layouts such as tabbed.